cartographer/mapping/id.h

```diff
@@ -322,6 +323,43 @@ class MapById {
 
   bool empty() const { return begin() == end(); }
 
+  ConstIterator lower_bound(int trajectory_id, const common::Time& time) {
```

Comment:
documentation. This is not an obvious function - especially the interplay between the two parameters. While you are on it: document behavior of this function, `BeginOfTrajectory` and `EndOfTrajectory` for non-existing trajectories.

Reply:
DOne